src/utils/webvtt-parser.js

```diff
@@ -139,17 +138,19 @@ const WebVTTParser = {
             }
           });
           try {
+            //
             // Calculate subtitle offset in milliseconds.
-            if (syncPTS + ((vttCCs[cc].start * 90000) || 0) < 0) {
-              syncPTS += 8589934592;
-            }
+            // if (syncPTS + ((vttCCs[cc].start * 90000) || 0) < 0) {
```

Comment:
doesnt this do some syncing for over 12 hrs of  running video ? may be check if thats this logic ?